docker-compose-dev.yml

```
@@ -60,6 +60,8 @@ services:
     volumes:
       - ./docker-dev-volumes/app/data:/dv
       - ./docker-dev-volumes/app/secrets:/secrets
+      # Map the glassfish applications folder so that we can update webapp resources using scripts/intellij/cpwebapp.sh
+      - ./docker-dev-volumes/glassfish/applications:/opt/payara/appserver/glassfish/domains/domain1/applications
```

```suggestion
      # Uncomment to map the glassfish applications folder so that we can update webapp resources using scripts/intellij/cpwebapp.sh
      # - ./docker-dev-volumes/glassfish/applications:/opt/payara/appserver/glassfish/domains/domain1/applications
```

Should we comment it out, like the example below?

---

Yes, perhaps. The only problem is that this file is checked into git and if users edit it, they may commit their (local) changes. This is another reason why we'd need a more modular docker compose approach in the future.

---

Sure, makes sense. I haven't uncommented the "./target" one in a few months (I haven't needed to hack on xhtml, thankfully) but it suffers from the same problem.

I vote we comment it out, merge it, update PR #9959 to include it. And then think more about it while we work on #9959.

---

@beepsoft I see the 👍 . Do you want to go ahead and click "accept suggestion" to get the commit in?